statscache/schedule.py

```diff
@@ -71,3 +72,40 @@ def __float__(self):
 
     def __int__(self):
         return int(float(self))
+
+
+class Frequency(object):
```

Comment:
It's not clear to me how this Frequency class is used.

- Can you add some tests for it next to the ``Schedule`` tests?